### ISSUE TITLE
ReLU after 1x1 conv

### ISSUE THREAD

**Issue by xuxu116** [huanghoujing/beyond-part-models#16]
I am curious about the relu function which you use after the conv. It will force the feature to be all positive values and could you tell me the reason for using relu in the code below? I found there is no explicit word in the origin paper for this part. Thx!
    for _ in range(num_stripes):
      self.local_conv_list.append(nn.Sequential(
        nn.Conv2d(2048, local_conv_out_channels, 1),
        nn.BatchNorm2d(local_conv_out_channels),
        nn.ReLU(inplace=True)
      ))

**Comment by huanghoujing** [comment]
Thank you for attention!

The last operation at the end of the last conv layer of ResNet is also ReLU, so I think it's proper to use ReLU here. Besides, it is common practice to use Conv+BN+ReLU.
